tools/flaky_test_monitor/common/testdata/test_data.go

```diff
@@ -19,24 +19,24 @@ type Level1TestData struct {
 }
 
 type Level2TestData struct {
-	Directory        string
-	Level1DataPath   string
-	HasFailures      bool
-	HasNoResultTests bool
-	Level1Summaries  []common.Level1Summary
+	Directory       string
+	Level1DataPath  string
+	HasFailures     bool
+	HasExceptions   bool
+	Level1Summaries []common.Level1Summary
 }
 
 // ************** Helper Functions *****************
 // following functions are used to construct expected TestRun data
 
 func getCommitDate() time.Time {
-	commitDate, err := time.Parse(time.RFC3339, "2021-09-22T01:06:25Z")
+	commitDate, err := time.Parse(time.RFC3339, "2021-09-21T18:06:25-07:00")
```

Comment:
what's the purpose of changing the time stamp? to make it more realistic where there will be an offset from UTC?